changed the way we handle the return value when not finding incidents (#22521)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_10_9.md

#### Scripts
##### SearchIncidentsV2

> - Fixed an issue where no incidents were found, but a missleading message of 'incidents found' was presented. 

Reviewer: ```suggestion
- Fixed an issue where a `incidents found` message was shown when no incidents were found.
```
